libbeat/docs/breaking.asciidoc

```diff
@@ -9,9 +9,155 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
 
 See the following topics for a description of breaking changes:
 
+* <<breaking-changes-6.3>>
 * <<breaking-changes-6.0>>
 * {auditbeat}/auditbeat-breaking-changes.html[Breaking changes in Auditbeat 6.2]
 
+[[breaking-changes-6.3]]
+=== Breaking changes in 6.3
+
+This section discusses the main changes that you should be aware of if you
+upgrade the Beats to version 6.3. Please also review the relevant
+Breaking Changes sections of the <<release-notes,release notes>>.
+
+[[breaking-changes-mapping-conflict]]
+==== New `host` namespace may cause mapping conflicts for Logstash
+
+This breaking change applies only to users who send Beats events to Logstash.
+
+Starting with version 6.3, Beats provides an `add_host_metadata` processor for
+adding fields, such as `host.name` and `host.id`, to Beats events. These fields
+are grouped under a `host` prefix and conform to the
+https://github.com/elastic/ecs[Elastic Common Schema (ECS)]. The `host` object
+is defined in the Elasticsearch index template even if the processor is not
+used.
+
+We've also added a `host.name` field to all events sent by Beats. This field
+prevents the Beats input plugin in Logstash from adding a default `host` field.
+(By default, the plugin adds a `host` field if the event doesn't already have
+one. We don't want the plugin to add this field because it causes a mapping
+conflict with the `host` object defined in the index template.)
+
+*What does this mean to you?*
+
+See the info for your particular use case:
+
+* <<beats-template-versioned-indices>>
+* <<custom-template-non-versioned-indices>>
+* <<beats-template-non-versioned-indices>>
+
+[[beats-template-versioned-indices]]
+===== Use case: You use the Beats index template and versioned indices
+
+In this use case, you load the versioned template manually and use the Beats
+versioned index pattern, `%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}`,
+as recommended in the {logstash-ref}/plugins-inputs-beats.html[Beats input
+plugin] documentation. This results in a `host` field in 6.2 indices, and a
+`host.name` field in 6.3 indices. There are no mapping conflicts. Cross index
+searches in Elasticsearch still work as expected. 
+
+*What do you need to change?*
+
+If you searched for the `host` field previously, you now need to search for the
+`host` and `host.name` fields.  
+
+If you have multiple visualizations in Kibana that reference the `host` field,
+you can export the objects by using the Kibana API or UI, change any aggregation
+references to use `host.name` instead of `host`, and then re-import the
+objects into Kibana.
+
+
+//TODO: Add links to the Kibana docs for exporting/importing objects.
```

Comment:
Want to resolve it as part of this PR?

Reply:
@ruflin If you think it's good enough to merge, maybe we should merge and then iterate over it after it's merged. I think @tsg wanted to review this.